db/redis_test.go

```diff
@@ -0,0 +1,166 @@
+package db
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/alicebob/miniredis/v2"
+	"github.com/vulsio/go-kev/models"
+	"github.com/vulsio/go-kev/utils"
+	"golang.org/x/xerrors"
+)
+
+func setupRedis() (*miniredis.Miniredis, DB, error) {
+	s, err := miniredis.Run()
+	if err != nil {
+		return nil, nil, xerrors.Errorf("Failed to run miniredis: %w", err)
+	}
+	driver, err := NewDB("redis", "redis://"+s.Addr(), false, Option{})
+	if err != nil {
+		return nil, nil, xerrors.Errorf("Failed to new db: %w", err)
+	}
+	return s, driver, nil
+}
+
+func teardownRedis(s *miniredis.Miniredis, driver DB) {
+	s.Close()
+	_ = driver.CloseDB()
+}
+
+func TestGetKEVByMultiCveID(t *testing.T) {
+	t.Parallel()
+	s, driver, err := setupRedis()
+	if err != nil {
+		t.Errorf("Failed to prepare redis: %s", err)
+	}
+	defer teardownRedis(s, driver)
+	if err := prepareTestData(driver); err != nil {
+		t.Errorf("Failed to prepare testdata of KEVulns: %s", err)
```

Comment:
The reason for changing from Errorf to Fatalf is the same as that for setup.
In the current implementation, `KEVuln` refers to CISA, but since we will be inserting CISA and VulnCheck data this time, I think it would be fine to use `KEV`.
```suggestion
		t.Fatalf("Failed to prepare testdata of KEV: %s", err)
```